ksqldb-streams/src/main/java/io/confluent/ksql/execution/streams/metrics/RocksDBMetricsCollector.java

```diff
@@ -87,20 +87,14 @@
   private static final Object lock = new Object();
 
   private static Map<String, Collection<AggregatedMetric<?>>> registeredMetrics = null;
-  private final Metrics metrics;
-
-  public RocksDBMetricsCollector() {
-    this(MetricCollectors.getMetrics());
-  }
-
-  @VisibleForTesting
-  RocksDBMetricsCollector(final Metrics metrics) {
-    this.metrics = Objects.requireNonNull(metrics);
-  }
```

Comment:
The metrics object is just set in `configure` now. All test and production code uses the default 0-arg constructor.